systemtest/src/test/java/io/strimzi/systemtest/upgrade/AbstractUpgradeST.java

```diff
@@ -379,21 +378,19 @@ protected void setupEnvAndUpgradeClusterOperator(BundleVersionModificationData u
                 ResourceManager.waitForResourceReadiness(getResourceApiVersion(KafkaTopic.RESOURCE_PLURAL), testStorage.getTopicName());
             }
 
-            String producerAdditionConfiguration = "delivery.timeout.ms=20000\nrequest.timeout.ms=20000";
+            String producerAdditionConfiguration = "delivery.timeout.ms=35000\nrequest.timeout.ms=35000";
```

Comment:
Again, you should deal with flaky tests in different PR
It's a bit confusing why are you changing the configuration here.